sidebars.js

```diff
@@ -445,10 +445,11 @@ module.exports = {
               },
               items: [
                 "client/javascript-mono/MigrationFromOldJsClient",
-                // "client/javascript-mono/MigrationFromOldReact",
+                "client/javascript-mono/MigrationFromOldReact",
```

Comment:
next change - have a folder just for migration, but then top level js, react, react native and a folder for the (now deprecate) statsig-js, statsig-react, statsig-react-native